File: SDKs/google.home/2.0/python/src/sdk/models/operations/pairwithspeaker.py
import dataclasses
from typing import Optional
from ..shared import pairwithspeakerrequest as shared_pairwithspeakerrequest


@dataclasses.dataclass
class PairwithSpeakerRequest:
    request: shared_pairwithspeakerrequest.PairwithSpeakerRequest = dataclasses.field(metadata={'request': { 'media_type': 'application/json' }})
    

@dataclasses.dataclass
class PairwithSpeakerResponse:
    content_type: str = dataclasses.field()
    status_code: int = dataclasses.field()
    pairwith_speaker_200_text_plain_object: Optional[str] = dataclasses.field(default=None)
    

File: SDKs/google.home/2.0/typescript/src/sdk/models/operations/pairwithspeaker.ts
import { SpeakeasyMetadata, SpeakeasyBase } from "../../../internal/utils";
import * as shared from "../shared";



export class PairwithSpeakerRequest extends SpeakeasyBase {
  @SpeakeasyMetadata({ data: "request, media_type=application/json" })
  request: shared.PairwithSpeakerRequest;
}


export class PairwithSpeakerResponse extends SpeakeasyBase {
  @SpeakeasyMetadata()
  contentType: string;

  @SpeakeasyMetadata()
  pairwithSpeaker200TextPlainObject?: string;

  @SpeakeasyMetadata()
  statusCode: number;
}
